components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,692 @@
+
+#### 3.3.3. Sea surface height gradient
+The sea surface height (SSH) gradient uses the standard gradient formulation from cell center to edge,
+
+$$
+-g\nabla(h_i-b_i) &= -g\frac{1}{d_e} \sum_{i\in CE(e)} -n_{e,i}(h_i-b_i)\\
+&= -g\frac{(h_{i2}-b_{i2}) - (h_{i1}-b_{i1}) }{d_e}.
+$$
+
+#### 3.3.4. Del2 momentum dissipation
+The Del2, or Laplacian operator, viscous momentum dissipation maps edge-normal velocity back to the edge-normal component of the Laplacian. In TRiSK this is done with the vorticity-divergence formulation, which may be written as
+
+$$
+\nabla^2 {\bf u} &=  \nabla \left( \nabla\cdot{\bf u}\right) - \nabla \times \left( \nabla \times {\bf u} \right) \\
+ &=  \nabla {\bf D} - \nabla \times \omega \\
+ &=  \nabla {\bf D} - {\bf k} \times \nabla \omega \\
+ &=  \nabla {\bf D} - \nabla^\perp \omega.
+$$
+This formulation is also mentioned in [Gassmann 2011](https://linkinghub.elsevier.com/retrieve/pii/S0021999111000325) (equation 17), [Gassman 2018](https://onlinelibrary.wiley.com/doi/10.1002/qj.3294) (equation 44) and [Lapolli et al. 2024](https://www.sciencedirect.com/science/article/pii/S1463500324000222) (section 4.5.2).
+
+For our discretization, the full Del2 term is written as
+$$
+\nu_2 \nabla^2 u_e &= \nu_2 \left( \nabla D_i - \nabla^\perp \omega_v\right) \\
+&= \nu_2 \left( \frac{1}{d_e} \sum_{i\in CE(e)} -n_{e,i}D_i - \frac{1}{l_e} \sum_{v\in VE(e)} -t_{e,v}\omega_v\right) \\
+&= \nu_2 \left( \frac{D_{i2} - D_{i1}}{d_e} - \frac{\omega_{v2} - \omega_{v1}}{l_e} \right),
+$$
+where the ordering of indices $\{i_i, i_2\}$ and $\{v_1, v_2\}$ are explained in the gradient operator sections above.
+
+An alternative formulation for the Del2 dissipation on an unstructured mesh is presented in section 4.2 and Appendix B of 
+[Gassman 2018](https://onlinelibrary.wiley.com/doi/10.1002/qj.3294).
+
+#### 3.3.5. Del4 momentum dissipation
+The Del4, or biharmonic, momentum dissipation also maps edge-normal velocity back to the edge-normal component of the Laplacian. This is done with two applications of the Del2 operator above.
+
+$$
+- \nu_4 \nabla^4 u_e = - \nu_4 \nabla^2 \left( \nabla^2 u_e \right)
+$$
+
+#### 3.3.6. Rayleigh Drag
+Rayleigh drag is simply linear drag, applied to all levels. It is typically only used during the spin-up process to damp large velocities during the initial adjustment. The Rayleigh coefficient $Ra$ is simply a scalar constant,
+
+$$
+- Ra \: u_e.
+$$
+
+#### 3.3.7. Bottom drag
+Bottom drag is more relevant to layered models than to shallow water systems, but is included here for completeness. It is a quadratic drag applied to the edge velocity,
+
+$$
+- C_D \frac{u_e\left|u_e\right|}{[h_i]_e}.
+$$
+
+#### 3.3.8. Wind forcing
```

Comment:
given this section I would suggest the generic form above and say we choose to model the wind stress as ...